index.html

```diff
@@ -10,21 +10,30 @@
 </head>
 
 <body>
-    <header class="white">
-        <a class="logo" href="#">My logo</a>
-        <nav class="nav-bar">
-            <div class="bars"><img src="images/menu.png" alt="menu navbar" /></div>
-            <div class="desk-menu">
-                <ul>
-                    <li>Portfolio</li>
-                    <li>About</li>
-                    <li>Contact</li>
-                </ul></div>
-        </nav>
+    <header class="menu-animation">
+        <div class="container">
+            <div class="logoforJS">
+               <h2>My logo</h2>
+            </div> 
+
+            <div class="nav-toggle" id="navToggle">
+                <img src="./images/menu.png" alt="hamburger-menu"/>
+            </div>        
+           
+            <nav class="desktop-nav">            
+                <ul class="links">
+                    <li class="close">X</li>                    

```

Comment:
- [ OPTIONAL] The hamburger and closed button must follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins). how about using a closed menu class icon, for example: `<i class="fa fa-close"></i>` instead of using a `li` tag with an `x` ? for more documentation you can check this [link](https://www.w3schools.com/icons/tryit.asp?filename=tryicons_fa-close) or you can download the icon image from template Figma design to make sure you are using the same icon with the original design